doc/interop-test-descriptions.md

```diff
@@ -1125,13 +1125,17 @@ registered custom LB policy. A load report reference can be passed from the call
 through, e.g. CallOptions, to receive metric reports.
 The test framework will fill in the reference with the latest load report from the report listener.
 This way, together with server behaviors we can verify the expected metric reports are received.
+Note that this test client includes multiple verification steps that requires the server to keep the states.
+Therefore, server side is required to have the capability of allowing only one client to perform the 
+`orca_oob` test case at a time, using a lock, see server features below.
 
 Server features:
 * [UnaryCall][]
+* [FullDuplexCall][]
 * [Backend Metrics Report][]
 
 Procedures:
-* Client sends a unary call to the server. The call request sets `orca_oob_report` to a test load report.
+1. Client starts a bidi-streaming call to the server. In the first message the request sets `orca_oob_report` to a test load report.
     ```
     {
       orca_oob_report:{
```

Comment:
It needs to request a response. Ditto below.

Reply:
This was marked resolved, but it doesn't seem resolved. This comment is talking about https://github.com/grpc/grpc-java/blob/69abec5a2881afe5525f45ffdbd8f4fb23cc53ff/interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java#L1798

Reply:
No, not in text. It is part of the message; you can include it in the message snippet. And it is also necessary in step 3, unless you remove the "After getting a response" in step 4.